src/pages/api/alternances/index.controller.test.ts

```diff
@@ -60,4 +64,22 @@ describe('rechercher une alternance', () => {
       page: 1,
     });
   });
+
+  describe('Quand les paramètres de l\'url ne respectent pas le schema de validation du controller', () => {
+    [
+      { page: '67' },
+      { page: '0' },
+      { page: 'NaN' },
+      { page: 'nan' },
+      { grandDomaine: 'CS12', page: '1' },
+      { page: '1', typeLocalisation: 'erreur' },
+      { codeLocalisation: 'erreur', page: '1', typeLocalisation: 'COMMUNE' },
+    ].forEach((queryParametersToTestInError) => {
+      it(`pour ${JSON.stringify(queryParametersToTestInError)} on retourne une erreur`, () => {
```

Comment:
Il y a qqch de built-in dans jest pour les loop de tests : https://jestjs.io/docs/api#testeachtablename-fn-timeout

Reply:
pas mal effectivement